docs/source/GETTING_STARTED.rst

```diff
@@ -0,0 +1,132 @@
+.. _GETTING_STARTED:
+
+Getting Started
+===============
+
+.. contents:: :local:
+
+Google Cloud Setup
+------------------
+
+.. warning:: Google Cloud Platform must approve several requests which may take up to 1 day to complete.
```

Comment:
The warnings and notes are not styled, just a plain-text "Warning" line on github.

I get that it looks good on RTD, but I hope there is a way for it to look good on github as well.

Reply:
My personal preference would be to ignore github related formatting issues for this pull request. The primary interaction point for this documentation should be on read the docs. Figuring out how to smooth over github related formatting issues seems like it should be a marked as an issue for a future pull request.

Reply:
I'm hoping we are ok with leaving these in place.